netaddr.go

```diff
@@ -545,17 +434,24 @@ func (ip IP) As4() [4]byte {
 // Note that unlike the Go standard library's IP.String method,
 // IP4-mapped IPv6 addresses do not format as dotted decimals.
 func (ip IP) String() string {
```

Comment:
I assume we'll come back and optimize this method more later.

Reply:
yeah, those are the commented out alloc TODOs in the test